### PR TITLE
Config REST Framework

### PR DESCRIPTION
**Pull request by pamella**
## Description

- [As a Programmer I write a RESTful endpoint to server the React Frontend](https://trello.com/c/L2RsHH49/26-as-a-programmer-i-write-a-restful-endpoint-to-server-the-react-frontend)

[Click here](https://trello.com/invite/b/Rh2vcXv4/f776d0c1add6f8df42e8c47ff9a4bc08/pokebattle-pamella) to join Pokebattle's trello card.

## Steps to reproduce (if appropriate):
In order to have access to pokemons list at local environment:
 ```
$ python manage.py shell
$ from pokemons.tasks import task_save_pokeapi_pokemons
$ task_save_pokeapi_pokemons.delay()
```

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires documentation updates.
- [ ] I have updated the documentation accordingly.
- [X] My change requires dependencies updates.
- [X] I have updated the dependencies accordingly.

### CODE REVIEW

File: battles/serializers.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            is_creator = team.trainer == obj.trainer_creator
+            role = 'creator_pokemon' if is_creator else 'opponent_pokemon'
+            rounds[0][role] = PokemonSerializer(team.pokemon_1).data
+            rounds[1][role] = PokemonSerializer(team.pokemon_2).data
+            rounds[2][role] = PokemonSerializer(team.pokemon_3).data
+
+        return rounds
+
+
+class TrainerTeamSerializer(serializers.ModelSerializer):
```
Comment (hugobessa) [inline battles/serializers.py]: Is this being used?

File: static/css/base.css [hunk 2]
```diff
@@ -1,10 +1,10 @@
 /* Common */
 html {
-    font-size: 16px;
+    font-size: 16px !important;
```
Comment (hugobessa) [inline static/css/base.css]: Why is `!imporatnt` necessary?

File: static/css/base.css [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
 }
 
 body {
-    font-family: 'Ubuntu', sans-serif;
+    font-family: 'Ubuntu', sans-serif !important;
```
Comment (hugobessa) [inline static/css/base.css]: Same here, why is `!imporatnt` necessary?

**Comment by pamella** [comment]
> @pamella I wrote a few comments.

@hugobessa  Done.

**Pull request merged**